.gitignore

```diff
@@ -63,4 +63,8 @@ scripts/generate-cpp-docs/c/docs
 
 # Generated datasets
 dataset/binary-demo
-dataset/databases/tinysnb
\ No newline at end of file
+dataset/databases/tinysnb
+
+#Database
+newDatabase
+history.txt
```

Comment:
Let's remove these changes to .gitignore. If you are testing via the shell, run the shell from outside your kuzu folder and the files won't show up in git.